v1 release: gatsby-transform-url

frederickfogerty opened the pull request.

sherwinski commented on packages/gatsby-transform-url/src/index.ts:
```diff
@@ -0,0 +1,148 @@
+     * The aspect ratio to set for the rendered image and the placeholder.
+     * Format: float. Can be calculated with ar = width/height.
+     */
+    ar: number;
```
should accept number and string

frederickfogerty commented on .travis.yml:
```diff
@@ -0,0 +1,30 @@
+  only:
+    - master
+    - next
+    - beta
```
It can only be removed after the PR is merged so I'll leave it for now.

frederickfogerty commented on packages/gatsby-transform-url/README.md:
```diff
@@ -0,0 +1,220 @@
+
+This library allows imgix urls to be used with gatsby-image. This library transforms imgix urls into a format that is compatible with gatsby-image. This can generate either fluid or fixed images. With this library you can either display images that already exist on imgix, or proxy other images through imgix.
+
+Unfortunately, due to limitations of Gatsby, this library does not support placeholder/blur-up images yet. When Gatsby removes this limitation, we plan to implement this for this library. Our other libraries will support blur-up/placeholder images.
```
Hmmm, I don't think this change captures what is intended here. By "our other libraries" I was referring to our other Gatsby libraries that are to be released. I can make this clearer here though.

frederickfogerty commented on packages/gatsby-transform-url/src/index.ts:
```diff
@@ -0,0 +1,148 @@
+  IImgixParams,
+} from './types';
+
+const VERSION = '1.0.0-beta.2';
```
Yeah, so `require`-ing from `package.json` doesn't "just work" in the browser. This requires either a Webpack or a Babel plugin to make this work. Since we don't use either of these, we can't be sure that this would work in the browser, so at this stage it has to stay like this.

frederickfogerty commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -0,0 +1,45 @@
+**To Reproduce**
+A bug is a _demonstrable problem_ that is caused by the code in the repository. Thus, the contributors need a way to reproduce your issue - if we can't reproduce your issue, we can't help you! Also, please be as detailed as possible.
+
+[a link to a codesandox or repl.it; here is a link to a codesandbox with imgix/gatsby installed which can be forked: https://codesandbox.io/s/imgix/gatsby-base-codesandbox-bhz8n]
```
Fascinating. I'll fix this up

frederickfogerty commented:
Hey @sherwinski, it looks like I've addressed all your concerns, so I'll merge this in, but please feel free to follow up if you see anything else that needs changing and I can fix that up in a PR 👍
